DayrSuryan/DSEthiop22.xml

```diff
@@ -79,17 +92,34 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                   <handDesc>
                      <handNote xml:id="h1" script="Ethiopic">
+                              <date notBefore="1950" notAfter="1951">Mid-20th-century script.</date>
                         <persName ref="PRS11944GabraM"/> 
-                        <!--same Gabra Māryām?-->
-                        <seg type="script">Written in large letters, the same hand as that of <ref type="mss" corresp="DSEthiop21"/>.</seg>
-                        <seg type="ink">Black, red.</seg>
+                              <seg type="ink">Black, red</seg>
+                              <desc>Careful but not very skilled hand; written in very
+                                 large (ca. 1 cm high) letters.</desc>
                      </handNote>
                   </handDesc>
+                  
+                  <additions>
+                     <list>
+                        <item xml:id="e1">
+                           <desc>Stamps of <placeName ref="INS0478DS"/>.</desc>
+                           <q xml:lang="ar"/>
+                        </item>
+                     </list>
+                  </additions>
                </physDesc>
 
                <history>
-                  <provenance>
-                  </provenance>
+               <origin>
+                  The manuscript is written in the same hand, same type of text block and binding as <ref type="mss" corresp="DSEthiop22"/>, 
+                  which according to its colophon was written by  <persName ref="PRS11944GabraM">Gabra Māryām of Māḫbara Səllāse</persName>
+                  for <ref target="INS0478DS"/>, in <date> 1951/52 CE</date>,
+                  in the tenure of <persName ref="PRS12955Tawfilus"/>,
+                  in <placeName ref="LOC6126WadinN">ʾAsqeṭəs</placeName>. It is therefore very probable that this manuscript was 
```

Comment:
the information "same type of text block and binding as" would do better as a separate sentence after "Asqetes".

Reply:
But the Asqetes part only refers to Ethiop. 21, it would make not much sense without the information on why it is relevant here?

Reply:
We can say smth like "copied in the same time and place as DSEthiop21", and specify in brackets (or even not - 21 will be just above).